.github/workflows/prow-github.yml

```
@@ -0,0 +1,36 @@
+permissions:
+  # Allow labeling issues
+  issues: write
+  # Allow adding a review to a pull request
+  pull-requests: write
+jobs:
+  prow-execute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpmcb/prow-github-actions@v2.0.0
+        with:
+          prow-commands: '/assign 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)